services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowReroutePayload.java

```diff
@@ -20,22 +20,22 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import lombok.Value;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.io.Serializable;
 
 /**
  * Flow reroute representation class.
  */
-@JsonSerialize
+// TODO move into api module
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
What abount `@JsonCreator` and json to properties mapping?

Reply:
This class becomes mutable for satisfying Mapstruct requiments. In that case, I haven't see any profit of using `@JsonCreator` constructor instead standard Jackson serialization\deserialization behavior with no-args constructor and field accessors